browser-extension/src/background.js

```diff
@@ -115,6 +122,11 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
     return true;
   }
 
+  if (request.message == 'addRateLaterBulk') {
+    addRateLaterBulk(request.videoIds).then(sendResponse).catch(sendResponse);
```

Comment:
Using `sendResponse()` to send an Error object does not behave as expected on Chrome.
Contrary to Firefox, Chrome uses JSON serialization when passing messages, and the `Error` object is lost.
See [Data Cloning Algorithm](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#data_cloning_algorithm) on MDN

Reply:
I opened https://github.com/tournesol-app/tournesol/pull/2069 to fix this.